test/testing.html

```diff
@@ -10,7 +10,7 @@
 <script type="text/javascript" src="../../../node_modules/steal/steal.js" main="@empty"></script>
 
 <!-- For this repo’s test suite -->
-<script type="text/javascript" src="../node_modules/steal/steal-sans-promises.js" main="@empty"></script>
+<!-- <script type="text/javascript" src="../node_modules/steal/steal-sans-promises.js" main="@empty"></script> -->
```

Comment:
You should put this back and just remove the `-sans-promises.js`. That is the default now in steal 2.x.